osgpkitools/incommon_request.py

```diff
@@ -204,10 +203,13 @@ def parse_args():
     arguments.update({'test': args.test})
     arguments.update({'login': args.login})
     
-    if args.usercert and args.userprivkey:
-        usercert, userkey = utils.find_user_cred(args.usercert, args.userprivkey)
-        arguments.update({'usercert': usercert})
-        arguments.update({'userprivkey': userkey})
+    if not args.usercert or not args.userprivkey:
+        raise InsufficientArgumentException("InsufficientArgumentException: " + \
+                                            "Please provide certificate(-c, --cert) and key(-k, --pkey) files\n")
+    
```

Comment:
Remember to remove the square brackets around -c cert -k pkey in the usage in incommon-request.py, and also in README.md.

Reply:
Removed.